WalletWasabi.Gui/Controls/WalletExplorer/ClosedWalletViewModel.cs

```diff
@@ -37,6 +37,22 @@ public ClosedWalletViewModel(Wallet wallet) : base(wallet)
 			}, this.WhenAnyValue(x => x.WalletState).Select(x => x == WalletState.Uninitialized));
 		}
 
+		public static WalletViewModelBase Create(Wallet wallet)
+		{
+			if (wallet.KeyManager.IsHardwareWallet)
+			{
+				return new ClosedHardwareWalletViewModel(wallet);
+			}
+			else if (wallet.KeyManager.IsWatchOnly)
+			{
+				return new ClosedWatchOnlyWalletViewModel(wallet);
+			}
+			else
+			{
+				return new ClosedWalletViewModel(wallet);
+			}
+		}
+
 		public ReactiveCommand<Unit, Unit> OpenWalletCommand { get; }
```

Comment:
This property should be moved above the `Create` method.
This might fix the CodeFactor issue.